### PR TITLE
Add new update json tests

### PR DESCRIPTION
**Pull request by JeffreyDallas**
Test result, all passed

https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1637175297101100

https://hedera-hashgraph.slack.com/archives/C018Y4E6ADT/p1637172943101000

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -956,6 +956,38 @@ workflows:
...
+          workflow-name: "GCP-Daily-Services-Comp-Update-Reconnect-Abort-4N-2C"
+          requires:
+            - build-platform-and-services
+          pre-steps:
+            - install-tools
+            - attach_workspace:
+                at: /
+
+  GCP-Daily-Services-Comp-Update-Reconnect-Abort-DisPreUpdate-4N-2C:
+    jobs:
```
Comment (Neeharika-Sompalli) [inline .circleci/config.yml:976]: need to add as a scheduled cron job , f.e
```
- schedule:
          cron: "30 6 * * *"
          filters:
            branches:
              only:
                - master

**Review by Neeharika-Sompalli — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -956,6 +956,52 @@ workflows:
...
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/4N_2C/UpdateReconnectAbortPreUpdate
```
Comment (qnswirlds) [inline .circleci/config.yml:995]: The change should not be here. It should be in regression repo. The result path should match the json description, so it should be `AbortDisPreUpdate`

Comment (JeffreyDallas) [reply .circleci/config.yml]: Updated. Regression side json also updated

File: .circleci/config.yml (hunk 1 above)
Comment (JeffreyDallas) [reply .circleci/config.yml:990]: scheduled added

**Review by qnswirlds — APPROVED** [review]

**Pull request merged**